ui/src/components/WalletStatus.tsx

```diff
@@ -102,29 +127,42 @@ export function WalletStatus() {
       {isAdmin && (
         <Card>
           <Flex direction="column" gap="3">
-            <Text size="4" weight="bold" color="red">Admin Controls</Text>
+            <Text size="4" weight="bold" color="red">
+              Admin Controls
+            </Text>
             <Text size="2" color="gray">
-              Transfer admin capabilities to another address. This action will remove your admin privileges.
+              Transfer admin capabilities to another address. This action will
+              remove your admin privileges.
             </Text>
-            
+
             <Flex direction="column" gap="2">
               <TextField.Root
-                placeholder="New admin address"
+                placeholder="Enter recipient address (0x...)"
```

Comment:
```suggestion
                placeholder="Enter new admin address (0x...)"
```

What do you think about this change?

Reply:
Yes, it is much better. I will update.